tests/php/_inc/lib/test_class.rest-api-endpoints.php

```diff
@@ -933,7 +933,7 @@ public function test_change_owner() {
 
 		// Create a user and set it up as current.
 		$user = $this->create_and_get_user();
-		$user->add_cap( 'jetpack_connect_user' );
+		$user->add_cap( 'jetpack_disconnect' );
```

Comment:
This was the wrong user cap to allow switching connection owner. Any user can connect_user, but not any user can own the connection.